src/main/java/org/generationcp/middleware/dao/ims/LotAttributeDAO.java

```diff
@@ -54,6 +55,23 @@ public List<AttributeDto> getLotAttributeDtos(final Integer lotId, final String
 		}
 	}
 
+	public List<LotAttribute> getLotAttributeValuesIdList(final List<Integer> lotIdList) {
+		List<LotAttribute> attributes = new ArrayList<>();
+		if (lotIdList != null && !lotIdList.isEmpty()) {
+			try {
+				final String sql = "SELECT {a.*}" + " FROM ims_lot_attribute a" + " WHERE a.lotId in (:lotIdList)";
```

Comment:
@cheryleigh should we exclude closed lots?

Reply:
Hi @clarysabel , checked and seems like this is only used for updating lots. Currently, we have a validation that only active lots can be updated, should we still join the lots table here and add the conditions to exclude them in this query?